prisma/schema.prisma

```diff
@@ -40,6 +40,8 @@ model Profile {
   linkedinUrl       String?
   githubLink        String?
   visaType          VisaType
+  companySize       CompanySizes[]
```

Comment:
Should this property name be plural, `companySizes` if it's an array? Also, it's not clear to me whether this is a profile of an individual or of a company. Will we have separate profile types for these entities, or is this one meant to cover both?

Reply:
Yeah it needs to be plural and an array. It will be multiple selection of checkboxes. It's for the user to select the sizes of companies to work for.